hr_timesheet_sheet/tests/test_hr_timesheet_sheet.py

```diff
@@ -398,12 +402,11 @@ def test_6(self):
         self.assertEqual(len(timesheet_3_4_and_5), 3)
 
         timesheet_6 = self.aal_model.create({
-            'name': 'z',
+            'name': empty_name,
```

Comment:
no

Reply:
Then we have significant inconsistency with test that was in v12 and I'm not sure what's the proper fix then

Reply:
And this line was '/' in v12